Kickstarter-iOS/Views/Controllers/AddNewCardViewController.swift

```diff
@@ -136,11 +141,22 @@ internal final class AddNewCardViewController: UIViewController,
     _ = self.zipcodeView
       |> \.autocapitalizationType .~ .allCharacters
       |> \.returnKeyType .~ .done
+
+    _ = self.reusableCardLabel
+      |> \.text %~ { _ in Strings.Remember_this_card() }
+
+    _ = self.reusableCardStackView
+      |> \.alignment .~ .center
+
+    _ = self.reusableCardSwitch
+      |> baseSwitchControlStyle
```

Comment:
Ideally we would only have 1 accessible element (the toggle - whose focus area would be the whole cell) but because we're faking a table view cell here we don't get that functionality...so because of that I think we should also add VoiceOver label on the switch that will simply duplicate the label but since it's a `UISwitch` it will also use `double tap to toggle` hint by default.

```suggestion
      |> baseSwitchControlStyle
      |> \.accessibilityLabel %~ { _ in Strings.Remember_this_card() }
```
(not sure if this will pass SwiftFormat)

Reply:
Also note that there seems to be an issue with `UISwitch` and right alignment..I've ran into it in my PR - https://github.com/kickstarter/ios-oss/pull/850

Basically even if you align right edges the switch still overlaps a little. Checked this out in view inspector.

<img width="708" alt="Screen Shot 2019-09-23 at 11 09 36 AM" src="https://user-images.githubusercontent.com/387596/65450803-d2cd3a00-ddf2-11e9-9452-bf2f49485efb.png">

Basically what helped was to align left edge of the switch to the right edge of superview minus the switch width.